src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx

```diff
@@ -74,3 +70,5 @@ export const MapPartygoerOverlay: React.FC<MapPartygoerOverlayProps> = ({
     />
   );
 };
+
+export const MapPartygoerOverlay = React.memo(_MapPartygoerOverlay);
```

Comment:
When using `React.memo` we have to ensure that the component is 'pure', with only changes in it's props causing changes any any of the child components below it. Is that the case for this component + all it's children? (eg. `UserProfilePicture`, etc)

If so, awesome! And then also I think the individual `useMemo`'s within it would be redundant/add unneeded overhead.